rpc/tests/rpc.rs

```diff
@@ -17,6 +18,140 @@ mod tests {
     };
     use std::str::FromStr;
 
+    fn get_test_tx() -> serde_json::Value {
```

Comment:
Can you remind me what the purpose of this function is? I see we're using it line 458 and 494 but it's unclear to me what we're testing

Reply:
`get_test_tx` is just a helper which returns a Kakarot transaction which I got from the devnet. This was previously hardcoded on line 458 and 494, and I updated the transaction to a Kakarot transaction and provided this helper to avoid boiler plate.

Reply:
Alright great